carma_cooperative_perception/src/multiple_object_tracker_component.cpp

```diff
@@ -112,7 +112,7 @@ auto make_detection(const carma_cooperative_perception_interfaces::msg::Detectio
       return make_ctra_detection(msg);
 
     case msg.MOTION_MODEL_CV:
-      break;
+      throw std::runtime_error("unsupported motion model type '3: constant velocity (CV)'");
```

Comment:
That is the default, but in practice we will probably change it through parameters in the launch configuration. I will change the default to be CTRV since that is supported.

We don't support CV models because the `multiple_object_tracking` library does not have them. We can add it in, but that would be a different issue.